vaadin-upload-flow-demo/src/main/java/com/vaadin/flow/component/upload/demo/UploadView.java

```diff
@@ -88,6 +91,31 @@ private void createSimpleUpload() {
                 output);
     }
 
+    private void createUploadWithFileConstraints() {
```

Comment:
@pleku I've added an example here that you can manually test here

As stated in the description 
 > `GeneratedVaadinUpload.FileRejectEvent` regenerated with detailError as a JsonString rather than an object. This has been merged in this PR https://github.com/vaadin/vaadin-upload-flow/pull/146/files